### PR TITLE
ci: add update website & format

### PR DESCRIPTION
**Pull request by sxzz**
Relate issue https://github.com/opensource-f2f/website/issues/29

### CODE REVIEW

File: .github/workflows/generator.yml [hunk 1]
```diff
@@ -2,14 +2,13 @@ name: generator
...
   push:
-    branches: [ master ]
+    branches: [master]
 
   workflow_dispatch:
 
 jobs:
   build:
     runs-on: ubuntu-latest
-    if: "!contains(github.event.head_commit.message, 'ci skip')"
```
Comment (sxzz) [inline .github/workflows/generator.yml:12]: Not necessary: https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs

**Comment by sxzz** [comment]
Please don't merge it before authorizing me the permission of setting the secret keys of the repo.

**Comment by LinuxSuRen** [comment]
hi @sxzz , thanks for helping on this. I just invite you to a maintainer role. Please let me know if it's not ok.

**Comment by sxzz** [comment]
@LinuxSuRen I still can't do that.
image

**Comment by LinuxSuRen** [comment]
hi @sxzz , already set you to the admin role. Please try it again.

**Comment by sxzz** [comment]
@LinuxSuRen OK already added keys. Ready to merge.

**Pull request merged**